src/logic/wallets/tests/ethAddresses.test.ts

```diff
@@ -0,0 +1,281 @@
+import {
+  isUserAnOwner,
+  isUserAnOwnerOfAnySafe,
+  isValidEnsName,
+  sameAddress,
+  shortVersionOf,
+} from 'src/logic/wallets/ethAddresses'
+import makeSafe from 'src/logic/safe/store/models/safe'
+import { makeOwner } from 'src/logic/safe/store/models/owner'
+import { List } from 'immutable'
+
+describe('sameAddress', () => {
```

Comment:
can we describe a little better what this is the intention of this suite? 
Only by reading "sameAddress" I can not guess what are these tests doing.

Reply:
perhaps something like `Utility or Logic: sameAddress function`